package.json

```diff
@@ -67,11 +72,16 @@
     "lerna": "^3.22.1",
     "markdown-link-check": "^3.8.1",
     "nock": "^10.0.6",
+    "openapi-types": "^11.0.1",
     "prettier": "^2.5.1",
     "rollup": "^2.22.2",
+    "swagger-typescript-api-nextgen": "^10.0.1",
```

Comment:
The author has been inactive since February. I'm uncertain why. There were a few features that weren't published yet, so I made the switch. I wanted sort types (broken) and auto loading the prettier config.

Reply:
I see. I'm a bit afraid that we will run into the same problem with the fork. I think if really needed we could always fork this out outself (should be fine since this is a dev dep), however I'm asking the question if we are choosing the right tool or if there are other more well-maintained alternatives that would suit us better.

Reply:
That is a good point, I'm leaving this unresolved to make others aware.

We talked about it, but I'll re-iterate for other reviewers:
All types that are generated have been moved into the `components.schemas` of the open api definition before running code generation. This should allow us to move to different libraries without issues. Ofcourse the way they may generate code (like prefering types for enums, or type comments may differ), but the future will tell if this becomes an issue.